pkg/controller/controller.go

```diff
@@ -0,0 +1,49 @@
+/*
+Copyright 2014 Google Inc. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package controller
+
+import (
+	framework "github.com/GoogleCloudPlatform/kubernetes/pkg/controller/framework"
+	"time"
+)
+
+// ControllerPlugin is an interface to controller plugins that can be run
```

Comment:
Moved and renamed.

NetworkPlugin doesn't follow this pattern, though its Host is named simply "Host".